config/jobs/image-pushing/OWNERS

```diff
@@ -4,8 +4,9 @@ options:
 
 # Oncallers with context on this
 reviewers:
-- Katharine
+- spiffxp
 
 # All oncallers
 approvers:
-- test-infra-oncall # oncall
+- test-infra-oncall
+- wg-k8s-infra-oncall
```

Comment:
None of the jobs under config/jobs/image-pushing run under the trusted cluster right?

Can we please fix this before we allow non-oncall to approve these PRs?

https://github.com/kubernetes/test-infra/blob/5d3fe432a2c4dba8361b974bc69f315abe2a7190/config/tests/jobs/jobs_test.go#L316-L338

Reply:
I think I did that below?

Reply:
I would like `TestTrustedJobs` to just validate the trusted jobs that test-infra-oncall approves.

I do not want to make this test more complicated (and therefore fragile) than necessary by making it validate other things (eg jobs in dir `D` use cluster `C`).

Reply:
Updated, PTAL